.travis.yml

```diff
@@ -12,6 +12,7 @@ before_script:
   - ./cc-test-reporter before-build
 script:
   - bundle exec rake
+  - bundle exec rubocop
```

Comment:
I might rather see rubocop run first so the failure is fast. Or, alternatively, handle this in the Rakefile (like https://github.com/sul-dlss/dlme-transform/blob/master/Rakefile#L7-L14). Don't feel too strongly about it, though.